CHANGELOG.md

```diff
@@ -1,7 +1,6 @@
 # cuML 0.17.0 (Date TBD)
 
 ## New Features
-- PR #3164: Expose silhouette score in Python
```

Comment:
I would not delete a PR entirely from the changelog. Just move this down to improvements and then have this PR listed below it for "Revert silhouette score exposure in python"